lib/storage/azure.js

```diff
@@ -495,9 +495,11 @@ module.exports = function() {
     getUrl: function (path) {
       /** @type {BlobSvc} */
       var blob = self.blobSvcs[0];
-      return blob.svc.getContainerClient(blob.container)
-        .getBlobClient(path)
-        .url;
+      const baseUrl = blob.svc.getContainerClient(blob.container)
```

Comment:
There's a nice feature built in to produce URLs for paths in azure, which @myovchev reasonably tried to use, but we do too many fussy things to paths ourselves, so fetch just a root path, remove the trailing slash per our expectations, and apply our standard path-adding function to avoid inconsistencies that break Apostrophe.